src/basic/PI_ATOMIC.cpp

```diff
@@ -66,5 +66,10 @@ void PI_ATOMIC::tearDown(VariantID vid)
   deallocData(m_pi);
 }
 
+size_t PI_ATOMIC::getBytesPerRep() const
+{
+  return (0*sizeof(Real_type) + 0*sizeof(Real_type)) * getRunSize();
```

Comment:
This will always return zero. Shouldn't it be 1*getRunSize(); i.e., the value for pi is read and written once per iteration?

Reply:
See my comment on the main thread about what to count in the case of reductions.